app/src/styles.sass

```diff
@@ -44,6 +44,13 @@ input
       background-color: tame($userbuttonbgcolor)
       color: tame($useboxfgcolor)
 
+.formactions.login
+  justify-content: center
+  height: 52px
+  padding-top: 14.5px
```

Comment:
Adding spaces above the button is a good idea. But maybe it would be better to use margin for that, instead of padding, because of the "collapsing" property of margins.